node/node.go

```diff
@@ -387,15 +386,15 @@ func (nc *Config) Build(ctx context.Context) (*Node, error) {
 	bservice := bserv.New(bs, bswap)
 	fetcher := net.NewBitswapFetcher(ctx, bservice, blkValid)
 
-	cstOffline := hamt.CborIpldStore{Blocks: bserv.New(bs, offline.Exchange(bs))}
+	ipldCborStore := hamt.CborIpldStore{Blocks: bserv.New(bs, offline.Exchange(bs))}
 	genCid, err := readGenesisCid(nc.Repo.Datastore())
 	if err != nil {
 		return nil, err
 	}
 
 	// set up chainstore
-	chainStore := chain.NewStore(nc.Repo.ChainDatastore(), bs, genCid)
-	chainState := cst.NewChainStateProvider(chainStore, &cstOffline)
+	chainStore := chain.NewStore(nc.Repo.ChainDatastore(), &ipldCborStore, genCid)
+	chainState := cst.NewChainStateProvider(chainStore, &ipldCborStore)
```

Comment:
In follow up, I think here and all the places below that get a chain store and cbor store separately should be able to use just the (wider) chain store interface now.